libs/shared-angular/src/lib/oral-attendance/oral-attendance.component.scss

```
@@ -0,0 +1,257 @@
+  border-radius: 50%;
+  background: var(--mat-sys-primary-container);
+  color: var(--mat-sys-on-primary-container);
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.decision-stamp {
+  position: absolute;
+  top: 2rem;
+  padding: 0.35rem 0.6rem;
+  border: 0.2rem solid currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)